src/token/stream-parser.ts

```diff
@@ -298,14 +293,14 @@ class Parser {
     });
   }
 
-  readBigUInt64LE(callback: (data: JSBI) => void) {
+  readBigUInt64LE(callback: (data: BigInt) => void) {
     this.awaitData(8, () => {
-      const low = JSBI.BigInt(this.buffer.readUInt32LE(this.position));
-      const high = JSBI.BigInt(this.buffer.readUInt32LE(this.position + 4));
+      const low = BigInt(this.buffer.readUInt32LE(this.position));
+      const high = BigInt(this.buffer.readUInt32LE(this.position + 4));
```

Comment:
Instead of doing the calculations ourselves here, I think we can just call out to [`readBigUInt64LE`](https://nodejs.org/docs/latest-v14.x/api/buffer.html#buffer_buf_readbiguint64le_offset).